Frameworks/UIKit/StarboardXaml/ApplicationMain.mm

```diff
@@ -106,12 +108,44 @@ int ApplicationMainStart(const char* principalName,
 
     [displayMode _updateDisplaySettings];
 
-    UIApplicationMainInit(principalClassName, delegateClassName, defaultOrientation, (int)activationType, activationArgument);
+    UIApplicationMainInit(principalClassName, delegateClassName, defaultOrientation, (int)activationType, activationArg);
     ScheduleMainRunLoop();
 
     return 0;
 }
 
+int ApplicationMainStartHSTRING(const char* principalName,
```

Comment:
I don't think we would be able to do that without making ApplicationMainStart take a void*, which we were trying to avoid

---
In reply to: [68821029](https://github.com/Microsoft/WinObjC/pull/592#discussion_r68821029) [](ancestors = 68821029)